### PR TITLE
Wang playground 2.0

### PR DESCRIPTION
**Pull request by J0hns0n-Wang**

### CODE REVIEW

File: anime/components/awardspage/ActualVoting.vue [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
-      <div v-for="r1 in r" class="nominee-box" @click="select">
+    <h1 class="award-name">{{ awardName }}</h1>
+    <div v-if="this.isAnime" class="nominee-container">
+      <div v-for="anime in animes" :key="anime" class="nominee-box" @click="select">
         <img class="image-placeholder" src="https://cdn.myanimelist.net/images/characters/4/457933.jpg" alt="">
-        <h1 class="anime-title">Anime Title {{r1}}</h1>
+        <h1 class="anime-title">{{anime.node.animeName}}</h1>
+      </div>
+    </div>
+    <!-- fix the v-ifs they are not working!!! -->
```
Comment (SelinaZheng) [inline anime/components/awardspage/ActualVoting.vue:10]: remember to delete this comment, the issue has been fixed

File: anime/components/awardspage/ActualVoting.vue [hunk 2]
```diff
@@ -23,16 +30,126 @@
 </template>
 
 <script lang="ts">
+import { useUserStore } from '~~/stores/userStore'
 export default ({
   data: () => ({
     r: [1,2,3,4,5,6],
     selected: false,
-    voted: true,
+    // voted: true,
```
Comment (harveyj123) [inline anime/components/awardspage/ActualVoting.vue:38]: use script setup

File: anime/components/awardspage/ActualVoting.vue
Comment (harveyj123) [inline anime/components/awardspage/ActualVoting.vue:1]: anqi use the things provided by vue

Comment (harveyj123) [reply anime/components/awardspage/ActualVoting.vue:1]: dont use js get element go read docs for vue. ref

Comment (harveyj123) [inline anime/components/awardspage/ActualVoting.vue:1]: use setup

**Review by harveyj123 — CHANGES_REQUESTED** [review]
change to setup

**Pull request merged**